config/initializers/sentry.rb

```diff
@@ -12,7 +12,7 @@
     # (there are lots and lots of status checks, and we want to know if there are
     # problems but we don't want to clog up Sentry with data)
     config.traces_sampler = lambda do |sampling_context|
-      /status/.match?(sampling_context[:env]["PATH_INFO"]) ? 0.01 : 0.1
+      /\A\/(status|health)\Z/.match?(sampling_context[:env]["PATH_INFO"]) ? 0.01 : 0.1
```

Comment:
question (not a blocker) - I think lower case 'z' would work, just as well. Why would we want a new line accepted in this regex?

> \z
> Matches the end of the string.
> \Z
> Matches the end of the string unless the string ends with a ``\n'', in which case it matches just before the ``\n''.

Reply:
Ah! That's me not having great regex-fu. It makes no practical difference (there will never be a newline character in the path), but `\z` is more appropriate here. Good spot, I will fix.